src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -36,7 +36,7 @@ public class AutomationIntegrationTests
         public AutomationIntegrationTests()
         {
             ValidationAppFolder = Path.GetFullPath(
-                Path.Combine(Directory.GetCurrentDirectory(), @"..\..\..\CurrentFileVersionCompatibilityTests\bin",
+                Path.Combine(Directory.GetCurrentDirectory(), @"../..\..\..\CurrentFileVersionCompatibilityTests\bin",
```

Comment:
For consistency, let's use forward slashes throughout
```suggestion
                Path.Combine(Directory.GetCurrentDirectory(), @"../../../../CurrentFileVersionCompatibilityTests\bin",
```